### PR TITLE
Removed <tt> from the tests, updated todo and updated .gitignore

### PR DESCRIPTION
**Pull request by amihaiemil**
Pull request for issue #16 .

### CODE REVIEW

**Comment by amihaiemil** [comment]
I removed the ``<<<fixed width font>>>`` from the file ``index.apt.vm`` . 
The root problem won't be solved until ``maven-site-plugin`` will create clean html5. 
I tried updating the jcabi-parent pom to the latest version which uses the latest version of ``maven-site-plugin (3.4)``, but nothing was changed. 

@dmarkov I also added the todo of ticket #6 since now more than 1 warning is produced.

**Comment by dmarkov** [comment]
@amihaiemil Let me find a reviewer for this pull request, thanks for submitting it

**Comment by dmarkov** [comment]
@pinaf review this one please

File: src/it/basics/verify.groovy [hunk 1]
```diff
@@ -64,22 +64,17 @@ MatcherAssert.assertThat(
...
-    Matchers.describedAs(htmlResponse.toString(), Matchers.hasSize(1))
+    Matchers.describedAs(htmlResponse.toString(), Matchers.hasSize(0))
 )
 MatcherAssert.assertThat(
     htmlResponse.warnings(),
     /**
      * @todo #86 This validation doesn't work because maven-site-plugin produces
-     *  invalid HTML5 output (it is using an obsolete NAME attribute on
-     *  some HTML elements). We're expecting exactly one warning here,
+     *  invalid HTML5 output (it is using obsolete attributes on
```
Comment (pinaf) [inline src/it/basics/verify.groovy]: @amihaiemil this puzzle is already live in #6 so let's keep it unchanged.

**Comment by pinaf** [comment]
@amihaiemil 1 comment

**Comment by amihaiemil** [comment]
@pinaf done.

**Comment by pinaf** [comment]
@rultor merge

**Comment by amihaiemil** [comment]
@yegor256 Can you confirm the merge if everything is ok? Thanks

**Comment by amihaiemil** [comment]
@yegor256 ping. This PR needs your approval to merge.

**Comment by yegor256** [comment]
@amihaiemil I didn't get the solution. you just removed the formatting, instead of fixing the problem or at least documenting it. we can as well remove the entire test and the build will be clean. huh? @pinaf please, pay attention

**Comment by amihaiemil** [comment]
@yegor256 I figured "don't test what's not supposed to be tested" since 

1) Doxia doesn't create clean HTML5..  and 
2) the HTML5 validator is "experimental" as shown on https://validator.w3.org/ underl ``Options``. So really, in my opinion, these tests don't have a solid basis to work on and should be removed entirely instead of leaving comments "don't use that element since tests will fail" . 

But on the other hand, it's really important that the page won't have errors so that's why I simply removed the causing element - I will write a comment about this.

**Comment by pinaf** [comment]
@yegor256 As @amihaiemil explained, the root problem is the unclean HTML5 generated by `maven-site-plugin`, which we don't control. I don't see any other solutions.

**Comment by amihaiemil** [comment]
@yegor256 I added the comment about the issue. Can you please merge if it's ok? Thanks.

**Comment by yegor256** [comment]
@rultor try to merge

**Comment by dmarkov** [comment]
@pinaf Thanks so much! Your account was topped up for **19 mins** (transaction ID is `AP-3RP10788Y02981018`, the task took 216 hours and 41 mins); review comments (c=4) added as a bonus; added +19 to your rating, now it is equal to [+6358](http://www.netbout.com/b/35128?open=rating)

**Comment by dmarkov** [comment]
@rultor deploy

**Pull request merged**